spec/cassia/response_handlers/connect_device_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+RSpec.describe Cassia::ResponseHandlers::ConnectDevice do
+  include FaradayHelpers
+  describe "#handle" do
+    it "returns true for a successful response" do
+      access_controller = Cassia::AccessController.new
+      response_body = {"status"=>"ok", "device"=>"F3:25:5F:22:35:39"}
+      response = build_response(status: 200, body: response_body)
```

Comment:
similar thing here, but can we add a space here and for all of the `response_handlers`?
Try to break it up with spaces between setup, action, and expectations